site/gatsby-site/server/emails/templates/NewIncident.ts

```diff
@@ -1,5 +1,4 @@
-exports = function () {
-  return `<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+export default `<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
@lmcnulty FYI -- César has proposed changes on which files to update. You should likely merge your PR after this merges to staging, and both of you collaborate on testing notifications in staging.

Reply:
(as I'm reviewing, reminder to @cesarvarela and @lmcnulty to sync on email template changes in addition to risk checklist API migration)